stormcloud/plugins/ms_update/ms_update_plugin.py

```diff
@@ -0,0 +1,72 @@
+import json
+import logging
+import os
+from typing import Any, List
+
+import boto3
+from create_ms_docs import SSTGeom, SSTMeta, SSTS3Document, SSTStart, SSTStats, create_ms_documents, sanitize_for_s3
+
+PLUGIN_PARAMS = {
+    "required": [
+        "watershed_name",
+        "transposition_domain",
+        "duration",
+        "s3_bucket",
+        "tropical_storm_json_s3_uri",
```

Comment:
Can we make this an optional parameter?

Reply:
These requests should be taken care of now